cmd/cli/main.go

```diff
@@ -81,6 +104,23 @@ var rollbackCmd = &cobra.Command{
 	Run: runRollback,
 }
 
+func getURL() string {
+	flags := []string{getAuthor, since, until, resource, namespace, name}
+	flagnames := []string{"author", "since", "until", "resource", "namespace", "name"}
+	url := "http://" + serverAddr + "/changes?"
+	qstn := false
+	for f, flag := range flags {
+		if flag != "" {
+			if qstn {
+				url += "&"
+			}
+			url += flagnames[f] + "=" + flag
+			qstn = true
+		}
+	}
+	return url
+}
+
```

Comment:
does the simpler version of getURL I rewrote earlier work?

`func getURL() string {
	flags := []string{author, since, until, resource, namespace, name}
	flagnames := []string{"author=", "since=", "until=", "resource=", "namespace=", "name="}
	var parts []string
	for i, flag := range flags {
		if strings.TrimSpace(flag) != "" {
			parts = append(parts, flagnames[i]+flag)
		}
	}
	url := "http://localhost:" + port + "/changes?"
	url += strings.Join(parts, "&")
	return url
}`